classes/PodsRESTHandlers.php

```diff
@@ -65,8 +66,16 @@ public static function get_handler( $object, $field_name, $request ) {
 		if ( empty( $pod_name ) ) {
 			$pod_name = pods_v( 'taxonomy', $object );
 		}
-
+		
+		if ( empty( $pod_name ) && 'attachment' == $object_type ) {
+			$pod_name = 'media';
+		}
 		$id  = pods_v( 'id', $object );
+
+		if ( empty( $id ) ) {
+			$id = pods_v( 'ID', $object );
+		}
```

Comment:
There's some more cases to cover but I'll make this more dynamic when I get a chance.